q2_fmt/_peds.py

```diff
@@ -237,6 +249,15 @@ def _compute_peds(peds_df: pd.Series, peds_type: str, peds_time: int,
                                          sample_row[reference_column],
                                          sample_row[subject_column],
                                          sample_row[time_column]]
+        if peds_type == "PPRS":
```

Comment:
this is also SO nitpicky and you can just ignore me if you want to - but there's a lot of inconsistent single vs. double quote usage. it does look nicer if that's uniform, but it's also not a critical change at all - take it or leave it!

Reply:
I tried to switch most of these to single quotes